packages/stylelint-config/src/index.js

```diff
@@ -55,6 +54,11 @@ module.exports = {
     // ReadMe breaks this rule in many places.
     'max-nesting-depth': null,
 
+    // Prefer neither "prefix" or "context" notations for media queries.
+    // e.g. "min-width: 400px" or "width >= 600px"
+    // https://stylelint.io/user-guide/rules/media-feature-range-notation/
+    'media-feature-range-notation': null,
```

Comment:
This rule was turned on by the latest stylelint-config-standard, but we still use both in our codebase. So keep this supporting both for now.